File: versioned_docs/version-1.14.x/examples/cpp/src/data_replication_remove.cc
#include <reduct/client.h>
#include <iostream>
#include <cassert>

using reduct::IBucket;
using reduct::IClient;
using reduct::Error;

int main() {
    // Create a client instance
    auto client = IClient::Build("http://127.0.0.1:8383", {
            .api_token = "my-token"
    });

    // Remove the `repl-to-remove` replication
    auto err = client->RemoveReplication("repl-to-remove");
    assert(err == Error::kOk);
}

File: versioned_docs/version-1.20.x/examples/go/src/data_replication_remove.go
package main

import (
	"context"
	reduct "github.com/reductstore/reduct-go"
)

func main() {
	// Create a client and use the base URL and API token
	client := reduct.NewClient("http://localhost:8383", reduct.ClientOptions{
		APIToken: "my-token",
	})

	// Remove the replication with the name "repl-to-remove"
	err := client.RemoveReplicationTask(context.Background(), "repl-to-remove")
	if err != nil {
		panic(err)
	}
}
